extra/selection_sort.cpp

```diff
@@ -0,0 +1,51 @@
+#include <iostream>
+using namespace std;
+#include <vector>
```

Comment:
Always do the work with preprocessor directives first, then do anything else.
```suggestion
#include <iostream>
#include <vector>

using namespace std;

```